views/confirmation.html

```diff
@@ -16,9 +16,10 @@
     <div class="govuk-grid-column-two-thirds">
 
       <div class="govuk-panel govuk-panel--confirmation">
-        <h1 class="govuk-panel__title">Application submitted</h1>
+        <h1 class="govuk-panel__title">{{ "Update statement submitted" if isUpdate else "Application submitted" }}</h1>
         <div class="govuk-panel__body">
-          Your application reference number: <br><strong>{{ referenceNumber }}</strong>
+          Your application reference number: <br>
+          <strong>{{ referenceNumber }}</strong>
```

Comment:
I know this is minor format change but if registration uses this view as well has it changed the view for registration as well? Do we want that? Maybe we want to use a separate view altogether now for registration? 👀  @stephanie-ch

Reply:
fine with conditionals here if Stephanie is too but just check this format change applies to Registration too and if thats ok with Matthew maybe?

Reply:
ignore meeee :D